content/en/docs/reference/kubectl/generated/kubectl_config/kubectl_config_set-cluster.md

```diff
@@ -309,5 +358,5 @@ kubectl config set-cluster NAME [--server=server] [--certificate-authority=path/
 
 ## {{% heading "seealso" %}}
 
-* [kubectl config](kubectl_config/)	 - Modify kubeconfig files
+* [kubectl config](../)	 - Modify kubeconfig files
```

Comment:
(nit / highlighting something).
We avoid relative links, but I think in generated docs it's fine.